pkg/graph/schema/types/model_to_operations_matrix.graphql

```diff
@@ -0,0 +1,149 @@
+  facilitatedBy: MTOFacilitator
+  status: SolutionStatus!
+  riskIndicator: MTORiskIndicator
+  commonSolutionID: UUID
+  solutionType: SolutionType! # If sourced from a Common Solution, this will resolve from the common solution table (you can't edit this for common solution!)
+
+  # Custom Resolvers
+  relatedMilestones: [Milestone!]! # Can be empty, since you can have a solution with no milestones
+  addedFromSolutionLibrary: Boolean!
+  commonSolution: CommonSolution
+}
+
+enum SolutionType {
+  IT_SYSTEM
+  CONTRACTOR
+  OTHER
+}
+
+enum MTOFacilitator {
+  MODEL_TEAM
+  MODEL_LEAD
+  IT_LEAD
+  SOLUTION_ARCHITECT
+  IT_SYSTEM_TEAM_OR_PRODUCT_OWNER
+  PARTICIPANTS
+  APPLICATION_SUPPORT_CONTRACTOR
+  IMPLEMENTATION_CONTRACTOR
+  EVALUATION_CONTRACTOR
+  QUALITY_MEASURES_DEVELOPMENT_CONTRACTOR
+  LEARNING_CONTRACTOR
+  MONITORING_CONTRACTOR
+  CONTRACTING_OFFICERS_REPRESENTATIVE
+  LEARNING_AND_DIFFUSION_GROUP
+  RESEARCH_AND_RAPID_CYCLE_EVALUATION_GROUP
+  OTHER
+}
+
+enum MTORiskIndicator {
+  ON_TRACK
+  OFF_TRACK
+  AT_RISK
+}
+
+# TODO Determine what these should actually be with Zoë & Natalie
+enum MilestoneStatus {
+  MILESTONE_STATUS_ONE
+  MILESTONE_STATUS_TWO
+}
+
+# TODO Determine what these should actually be with Zoë & Natalie
+enum SolutionStatus {
+  SOLN_STATUS_ONE
+  SOLN_STATUS_TWO
+}
+
+# CommonMilestone represents a Milestone from the "Common Milestones" library
+# It should not be used in contexts where a milestone has been added to a Model Plan (at that point, it'll be a "Milestone", not a "CommonMilestone")
+type CommonMilestone {
+  # DB Fields
+  id: UUID! # TODO Is this really needed? It might be kind of confusing to reference `id` like this
+  key: CommonMilestoneKey! # Similar to OperationalSolutionKey, enum of possible common milestones
+  name: String!
+  description: String!
+
+  # Custom Resolvers
+  isAdded: Boolean! # To represent if this Common Milestone is _already_ part of the Model Plan's MTO
+  isSuggested: Boolean!
+  commonSolutions: [CommonSolution!]!
+}
+
+type CommonSolution {
+  # DB Fields
+  id: UUID! # TODO Is this really needed? It might be kind of confusing to reference `id` like this
+  key: CommonSolutionKey! # Similar to OperationalSolutionKey, enum of possible common milestones
+  name: String!
+  description: String!
+
+  # Custom Resolvers
+  isAdded: Boolean! # To represent if this Common Solution is _already_ added of the Model Plan's MTO (to _any_ Milestone)
+  isSuggested: Boolean!
+}
+
+# TODO Update these with real values
+# Right now, this is just a placeholder type to represent the possible common milestones
+enum CommonMilestoneKey {
+  MILESTONE_A
+  MILESTONE_B
+}
+
+# TODO Update these with real values
+# Right now, this is just a placeholder type to represent the possible common solutions
+enum CommonSolutionKey {
+  SOLUTION_ONE
+  SOLUTION_TWO
+}
+
+# TODO: Add a way to determine order of Categories and subcategories
+type Category {
```

Comment:
I just realized we may want to call this MTOCategory as the type (to avoid any name conflicts in the future. I can update when building out the category table though!